compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/Names.java

```diff
@@ -35,7 +35,9 @@
     public static final Name BUILTIN_ORG = new Name("ballerina");
     public static final Name RUNTIME_PACKAGE = new Name("runtime");
     public static final Name TRANSACTION_ORG = new Name("ballerina");
+    public static final Name STREAMS_ORG = new Name("ballerina");
     public static final Name TRANSACTION_PACKAGE = new Name("transactions");
+    public static final Name STREAMS_PACKAGE = new Name("streams");
```

Comment:
Shall we rename this variable to "STREAMS_MODULE" since we are moving away from "package" now?